exercises/robot-simulator/src/lib.rs

```diff
@@ -0,0 +1,32 @@
+#[derive(PartialEq, Debug)]
```

Comment:
Seems good. It makes me wonder that if we are going to be in the habit of providing stubs, maybe this comment should be a standard that goes on top of all of them...

clearly an idea for another PR, but neverthelss worth bringing up while we are gthinking about it here